submissions/Liddell777/hooli-style-popup/style.css

```diff
@@ -196,27 +204,33 @@ header {
     margin: 40px 0;
 }
 
-.mainpopup__item-label:hover {
+.main-popup__item-label:hover {
+    font-size: 0px;
+    background-color: #eb665d;
+}
+.main-popup__item-label:hover::after {
+    font-size: 16px;
+    content: "less";
     background-color: #eb665d;
 }
 
-.mainpopup__item-label:before {
+/* .main-popup__item-label:before {
     content: "More";
-}
+} */
 
-.mainpopup--invisible {
+.main-popup--invisible {
     visibility: hidden;
 }
 
-.mainpopup__link--hidden {
+.main-popup__link--hidden {
     display: none;
 }
 
-input.menu__input:checked ~ div.mainpopup--invisible {
+input.menu__input:checked ~ div.main-popup--invisible {
```

Comment:
Avoid creating too specific CSS selectors.
[Read more about it](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).
In this case selector without HTML tags will work fine as current but faster)